php-fpm/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM registry.service.opg.digital/opguk/nginx
 
-ARG PHP_PACKAGES="php5-fpm php5-cli php5-xdebug php5-xhprof php5-imagick php5-mongo mongodb-clients php5-curl php5-xsl postgresql-client php5-pgsql php5-mcrypt"
+ARG PHP_PACKAGES="php5-fpm php5-cli php5-xdebug php5-xhprof php5-imagick php5-mongo mongodb-clients php5-curl php5-xsl postgresql-client php5-pgsql php5-mcrypt mcrypt"
 
```

Comment:
it is related  in that the apt commands that are removed from the dockerfiles for the app are now part of the base fpm build